src/applications/claims-status/components/claim-overview-tab/ClaimPhaseStepper.jsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Link } from 'react-router-dom-v5-compat';
+import { buildDateFormatter } from '../../utils/helpers';
+
+export default function ClaimPhaseStepper({
+  claimDate,
+  currentClaimPhaseDate,
+  currentPhaseStep,
```

Comment:
I think it might make sense for us to pick either "phase" or "step" internally, it seems weird that we are using these together in the same variable names when they refer to the same thing

Reply:
Agree I was kinda thinking the same thing...ill update and replace step with phase